client/data/site-migration/use-migration-enabled.ts

```diff
@@ -34,6 +33,7 @@ export const useMigrationEnabledInfoQuery = (
 		},
 		enabled: !! ( enabled && targetSiteId && sourceSite ),
 		retry: false,
+		refetchOnWindowFocus: false,
```

Comment:
I think leaving `refetchOnWindowFocus` as `true` here might be useful. Imagine you're on the migrate ready screen and for some reason Jetpack connection is broken during other manipulation. If we leave this on then it'll refetch and get the latest status. This is just a minor suggestion and probably not the use case for most of the time, so it's not a blocker here.

Reply:
Thanks for the suggestion; it makes sense.
If you don't mind, I'll go with this PR as it is since if I turn off the window focus re-fetching, I should probably improve the way of handling the loading state.

I'll leave it for the next PRs.